android/views/src/main/kotlin/com/boswelja/ephemeris/views/CalendarPagerAdapter.kt

```diff
@@ -49,50 +48,106 @@ internal class CalendarPagerAdapter : InfinitePagerAdapter<CalendarPageViewHolde
 
     override fun onBindHolder(holder: CalendarPageViewHolder, page: Int) {
         val pageState = pageLoader!!.getPageData(page)
-        holder.bindDisplayRows(dayBinder!!, pageState)
+        holder.bindDisplayRows(pageLoader!!, dayBinder!!, pageState)
```

Comment:
is this still needed after `pageLoader!!.getPageData(page)`? Is it not smart casting?

Reply:
pageLoader is mutable, therefore could change at any time (effectively impossible in this case though)